iml-gui/crate/src/components/command_modal.rs

```diff
@@ -577,44 +636,40 @@ fn extract_uri_id<T: EndpointName>(input: &str) -> Option<u32> {
     })
 }
 
-fn to_load_cmd(model: &Model, cmd_id: u32) -> bool {
-    // load it if the command is not found or is not complete
-    if let Some(cmd) = model.commands.get(&cmd_id) {
-        !cmd.complete
+fn perform_click(select: &mut Select, id: TypedId) -> bool {
+    if select.contains(id) {
+        !select.0.remove(&id)
```

Comment:
I don't understand why `select.contains()`, but `select.0.remove()`.

Reply:
Okay, `select.contains()` used in many places, it is more convenient to have this `contains` wrapper function.
`insert` and `remove` used only once, so it is not much sense to create their wrappers.

I solved this non-perfection by moving `perform_click ` function into `Select` itself.